src/contribute/overview/README.md

```diff
@@ -0,0 +1,62 @@
+# Contributing
+
+## Overview
+Welcome, and thanks for having an interest in contributing to the Huff Language and related repositories!
+
+All contributions are welcome! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+
+Contributing to Huff Language is not just limited to writing code. It can mean a wide array of things, such as participating in discussions in our [discord server](https://discord.gg/h8pkspwx) or reporting issues.
```

Comment:
Let's add a blurb above that all contributions are welcome for all huff-language ecosystem repositories, with `huff-rs` being the primary repository for Huff Lang syntax + compilation.